backend/src/common/client/api_server/v1/healthz_client.go

```diff
@@ -17,6 +17,7 @@ package api_server
 import (
 	"fmt"
 
+	"github.com/go-openapi/runtime"
```

Comment:
Should we add an alias to this import to avoid confusion with the variables `runtime` inside the `NewKubeflowInClusterHealthzClient` and `NewHealthzClient` functions?

Reply:
Sounds good. I will do it in another PR because other clients need to be updated as well.